src/Search.css

```diff
@@ -0,0 +1,64 @@
+@import url('https://fonts.googleapis.com/css?family=Itim');
+
+input[type=search]{
+   -moz-appearance: none;
+   -webkit-appearance: none;
+   appearance: none;
+}
+
+.search-div {
+  height: 50px;
+}
+
+.search-bar {
+  width: 40%;
+  padding: 2px 2px 2px 5px;
+  border-radius: 10px;
+  margin: 15px 30px 15px 15px;
+  font-size: 1.2rem;
+  font-family: Itim, Arial, Sans-Serif;
+  border-color: #00c7d6;
+  box-shadow: gray 2px 2px 4px;
+  text-shadow: gray 1px 1px 1px;
+  transition: transform .4s linear;
+}
+
+.search-bar:focus {
+  outline: none;
+  transform: scale(1.1);
+}
+
+
+button {
```

Comment:
Your button element would be a great place to add some sass by nesting your psuedo selectors for `hover`, `focus` and `active`.